File: T.java
public class T{
  public static void main(String[] args) {
    String [] names = {"FormDefine", "Form", "FormCategory", "FormInstance", "FileItem"};
    for(String it : names){
      System.out.println("Query<" + it + "> query" + it + "();");
    }
  }
}

File: t.js
var sys = require('sys');

var names = ["FormDefine", "Form", "FormCategory", "FormInstance", "FileItem"];
for(var i=0; i<names.length; i++) {
    var it = names[i];
    sys.puts("Query<" + it + "> query" + it + "();");
}
